pyuvsim/simsetup.py

```diff
@@ -324,22 +338,24 @@ def create_mock_catalog(time, arrangement='zenith', array_location=None, Nsrcs=N
         time (float or astropy Time object): Julian date
         arrangement (str): Point source pattern (default = 1 source at zenith).
             Accepted arrangements:
-                |  `triangle`:  Three point sources forming a triangle around the zenith
-                |  `cross`: An asymmetric cross
-                |  `zenith`: Some number of sources placed at the zenith.
-                |  `off-zenith`:  A single source off zenith
-                |  `long-line`:  Horizon to horizon line of point sources
-                |  `hera_text`:  Spell out HERA around the zenith
-                |  `random`:  Randomly distributed point sources near zenith
+            |  `triangle`:  Three point sources forming a triangle around the zenith
+            |  `cross`: An asymmetric cross
+            |  `zenith`: Some number of sources placed at the zenith.
+            |  `off-zenith`:  A single source off zenith
+            |  `long-line`:  Horizon to horizon line of point sources
+            |  `hera_text`:  Spell out HERA around the zenith
+            |  `random`:  Randomly distributed point sources near zenith
```

Comment:
This doesn't render properly on RTD: https://pyuvsim.readthedocs.io/en/code-format/classes.html#pyuvsim.simsetup.create_mock_catalog

Reply:
I've tried to change all of these to quoted lists.